client/src/pages/Login.js

```diff
@@ -544,27 +544,25 @@ const Login = ({ isLoginForm, forgotPassword, isAuthenticated }) => {
                     </p>
                   </>
```

Comment:
changes below are actually `eslint` formatting, only spacing changed but `git diff` is showing it this way, so below changes are all good